testing/runner.ts

```diff
@@ -37,9 +38,10 @@ export class Runner {
     try {
       // We tell the runner to start running at the end of current block of
       // synchronously executed code. This will typically be after all the
-      // suite definitions are evaluated. This is equivelant to setTimeout(..., 0).
+      // suite definitions are evaluated. This is equivalent to setTimeout(..., 0).
       await promisify(process.nextTick)();
       const ctx: IRunContext = {
+        testNameMatcher: new RegExp(process.env.TESTBRIDGE_TEST_ONLY) || /.*/,
```

Comment:
Small thing, I don't care too much - but if this makes a suite empty, we'll still run before and afters for the suite. You could possibly solve this by adding the filtering the actual `test()` method, so that these tests never even get registered, and change it so that suites with no tests just don't run anything (and also not run before afters).

Reply:
I couldn't find any nice way of doing this TBH, so leaving it for now.

If I do it somewhere around `test()`, I need to futz around in `globalStack` to figure out the current test path. If I do it instead at runtime, I need to recursively calculate all the tests that will run, before they run - which means constructing new `IRunContext` objects all the way down the Suite tree, so it becomes a bit of a mess.